#add-maintainer-mode.patch by Loic Minier <lool@dooz.org>
Index: alsa-lib-1.0.22/configure.in
===================================================================
--- alsa-lib-1.0.22.orig/configure.in	2010-01-17 16:31:14.000000000 +0100
+++ alsa-lib-1.0.22/configure.in	2010-01-17 16:33:04.000000000 +0100
@@ -22,6 +22,8 @@
 
 AC_PREFIX_DEFAULT(/usr)
 
+AM_MAINTAINER_MODE
+
 dnl Checks for programs.
 
 dnl try to gues cross-compiler if not set
